README.md

```diff
@@ -80,34 +96,35 @@ mkdir -p /tmp/rstudio-server
 
 Then launch the server similar to the docker command but with additional configuration options required:
 
+Set a password for your RStudio server:
+```
+RSERVER_PASSWORD=$(openssl rand -base64 15)
+echo $RSERVER_PASSWORD
+```
+Please notedown this RSERVER_PASSWORD as you will need when you log on to RStudio from a browser.
 ``` 
-Gene $RSERVER_PASSWORD
-
-
 PASSWORD=$RSERVER_PASSWORD singularity exec \
-	-B $(pwd):/home/rstudio/
+    -B $(pwd):/home/rstudio/
     -B /tmp/rstudio-server:/var/lib/rstudio-server \
     -B /tmp/rstudio-server:/var/run/rstudio-server \
     rnaseq-rstudio:4.1.0 \
-    rserver --auth-none=0 --auth-pam-helper-path=pam-helper --server-user ubuntu
-	
+    rserver --auth-none=0 --auth-pam-helper-path=pam-helper --server-user rstudio
 ```
 
-* `PASSWORD='yourpassword' singularity exec` sets the password environment variable and then runs the `singularity exec` command.
+* `PASSWORD=$RSERVER_PASSWORD singularity exec` sets the password environment variable and then runs the `singularity exec` command.
 * `-B $(pwd):/home/rstudio/` will mount a required wrtieable directory in the container. `pwd` is your current working folder. You can substitute any approriate directory for this.
 * `-B /tmp/rstudio-server:/var/lib/rstudio-server` and `-B /tmp/rstudio-server:/var/run/rstudio-server` mount additional required writeable directories.
 * `rstudio_4.1.0.sif` is the Singularity image file we built in the previous step.
-* `rserver --auth-none=0 --auth-pam-helper-path=pam-helper --server-user ubuntu` executes the command in the container, in this case "rserver" with various options. A key option here is the `--server-user ubuntu` which assumes the user on the host machine is called **ubuntu**, this is necessary with most Singularity setups to maintain the security mapping between users of the host and in the container.
-
-Assuming you are running Singularity on a headless remote server, you must ensure you connect to the host with port forwarding enabled if you wish to view the RStudio session on your local machine.
+* `rserver --auth-none=0 --auth-pam-helper-path=pam-helper --server-user ubuntu` executes the command in the container, in this case "rserver" with various options. A key option here is the `--server-user rstudio` which assumes the user on the host machine is called **rstudio**, this is necessary with most Singularity setups to maintain the security mapping between users of the host and in the container.
 
+Assuming you are running singularity on a headless remote server, you must ensure you connect to the host with port forwarding enabled if you wish to view the RStudio session on your local machine.
```

Comment:
Same comment as per the docker command. Simplify for your audience and use less jargon where possible please.

Reply:
Also does this need to be in its own section? Why would I need to do this instead of just running the command above?

Reply:
I am not too sure, what you mean here! 

Can you please change this as appropriate.